embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -44,11 +44,38 @@ while true; do
     esac
 done
 
+embulk_args="$@"
+
+while [ $# -gt 0 ] ; do
+    case "$1" in
+        "-b" | "--bundle")
+            shift
+            export EMBULK_BUNDLE_PATH="$1"
+            shift
+            break
+            ;;
+        *)
+            shift
+            ;;
+    esac
+done
+
+if test -z ${EMBULK_BUNDLE_PATH}; then
+    unset EMBULK_BUNDLE_PATH
+    unset BUNDLE_GEMFILE
+    export GEM_HOME="$(cd && pwd)/.embulk/jruby/$(java -cp $0 org.jruby.Main -e 'print RbConfig::CONFIG["ruby_version"]')"
```

Comment:
Let's use backquote instead of `$()`.
The`$()` is `bash` extension. 
Some OSes doesn't use `bash` as `/bin/sh` 
(As far as I remember, debian, ubuntu, xxBSD).

Maybe you can write `$HOME` instead of `$(cd && pwd)`.

```bash
GEM_HOME="$HOME/.embulk/jruby/`java -cp $0 org.jruby.Main -e 'print RbConfig::CONFIG["ruby_version"]'`"
```